graylog2-web-interface/src/components/lookup-tables/Cache.jsx

```diff
@@ -26,7 +27,12 @@ const Cache = React.createClass({
     return (
       <Row className="content">
         <Col md={6}>
-          <h2>{cache.title} <small>({plugin.displayName})</small></h2>
+          <h2>
+            {cache.title}
+            <ContentPackMarker contentPack={cache.content_pack} marginLeft={5} />
```

Comment:
In that case we would need a different component name, maybe something like `EntityTitle` or something like that, where the `ContentPackMarker` is just another thing to be included. Otherwise it would feel odd to me, having a `ContentPackMarker` component which takes the title and other information.